src/index.html

```diff
@@ -7,6 +7,74 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="
+        month
+        month--month-length--29
+        month--start-day--fri
+      ">
+        <div class="month__day"></div>
+
+        <div class="month__day"></div>
```

Comment:
Add empty line if there _multiline_ siblings. So if it's _oneline_ - there is no need